src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/importer/LocationsSpreadsheet.kt

```diff
@@ -46,11 +45,11 @@ class LocationsSpreadsheet(private val spreadsheet: Workbook, private val locati
         val locationType = LocationType.map(cells[TYPE].stringCellValue.toUpperCase().trim())
                 ?: throw IllegalArgumentException("Unsupported location type: " + cells[TYPE].stringCellValue)
 
-        val agency = cells[AGENCY].stringCellValue.toUpperCase().trim().takeUnless { it == "" }
-                ?: throw IllegalArgumentException("Agency id cannot be blank")
+        val agency = cells[AGENCY].stringCellValue.toUpperCase().trim().takeUnless { it.isBlank() }
```

Comment:
Minor (personal view), I think I would either have used `!!` to force an exception on null or used an IllegalArgument/State type exception rather than throwing a NPE manually.

Though I'm sure you end up with the correct behaviour either way, so no big deal